list_of_standing_committees_and_working_groups.md

```diff
@@ -0,0 +1,30 @@
+# List of Standing Committees and Working Groups
+
+Project work that expands beyond software-related work is organized into chartered domains and is carried out by Standing Committees and Working Groups. A description of these bodies and their function can be found [here](standing_committees_and_working_groups.md). This document contains a list of the current Standing Committees and Working Groups with charter summaries and links to their GitHub repositories.
+
+## Standing Committees
+
+### Diversity, Equity, and Inclusion
+
+Charter Summary: Increase diversity, equity and inclusion within all aspects of Project Jupyter, including the various councils/leadership, Subprojects, and the broader community.
+
+### Code of Conduct Incident Response
+
+Charter Summary: Own and manage the Jupyter Code of Conduct reporting and incident response processes and make determinations on individual incidents.
+
+### Conflict of Interest
+
+Charter Summary: Ensure that the joint and conflicting interests of leaders across Project Jupyter are disclosed appropriately, handled with transparency, and mitigated with integrity.
+
+## Working Groups
+
+### Trademark and Branding
+
+Charter Summary: License, protect, and promote the trademarks and visual and textual branding of Jupyter.
+
+## Community Building and Events
```

Comment:
Looks like this needs one more heading `#` to be under the working groups heading.
```suggestion
### Community Building and Events
```